script/test.py

```diff
@@ -24,17 +25,30 @@ def main():
     electron = os.path.join(SOURCE_ROOT, 'out', config,
                               '{0}.app'.format(PRODUCT_NAME), 'Contents',
                               'MacOS', PRODUCT_NAME)
+    resources_path = os.path.join(SOURCE_ROOT, 'out', config,
+                                   '{0}.app'.format(PRODUCT_NAME), 'Contents',
+                                   'Resources')
   elif sys.platform == 'win32':
     electron = os.path.join(SOURCE_ROOT, 'out', config,
                               '{0}.exe'.format(PROJECT_NAME))
+    resources_path = os.path.join(SOURCE_ROOT, 'out', config)
   else:
     electron = os.path.join(SOURCE_ROOT, 'out', config, PROJECT_NAME)
+    resources_path = os.path.join(SOURCE_ROOT, 'out', config)
 
+  use_instrumented_asar = '--use-instrumented-asar' in sys.argv
   returncode = 0
   try:
+    if use_instrumented_asar:
+      install_instrumented_asar_file(resources_path)
```

Comment:
This change causes `npm test` to use the instrumented `.asar` file so code coverage can be tracked.